cfgen/cfgen

```diff
@@ -112,6 +112,22 @@ profiles:  # This section is optional.  If it is missing, a single "default"
            # profile referring to all pools will be created.
   - name: <str>
     pools: [ <str> ]
+
```

Comment:
Note: This fragment could also go to the PR description.

Reply:
I've mentioned how to get for the format format in the commit message. Will copy-paste and add it anyway.